test/xpu/test_modules_xpu.py

```diff
@@ -90,7 +90,7 @@ def fn_to_gradcheck(*flat_input_and_params):
 @wrapSwapTensorsTest()
 def _test_to(self, device, dtype, module_info, training, swap, set_grad):
     module_cls = module_info.module_cls
-    devices = ['xpu']
+    devices = ['cpu', 'xpu']
```

Comment:
What's the meaning of adding `cpu` to `devices`?

Reply:
the case is to compare cpu and xpu op output, we aligned with cuda. If didn't add "cpu" , the case will error.

Reply:
It is ok to align with cuda but why the test will fail if no cpu in the devices list?

Reply:
because the order of testing will affect the results